src/main/java/org/example/BasePage.java

```diff
@@ -0,0 +1,48 @@
+package org.example;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.interactions.Actions;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.PageFactory;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+
+import java.time.Duration;
+
+public class BasePage {   // общий класс
+
+    WebDriver pageDriver = null;
```

Comment:
I recommend: make the variables protected, because protected access gives the subclass a chance to use the helper method or variable, while preventing a nonrelated class from trying to use it.

```
    protected Actions actions;
    protected WebDriverWait wait;
    protected WebDriver driver;
```